person.rb

```diff
@@ -0,0 +1,17 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name: 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that only the `Getter` is defined on the `:id` as it should only be able to be retrieved. Also you should ensure that it is present in the list of the **instance variable** such as in the example below.
```ruby
  ...
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
  ...
```